tests/functional/z_cluster/test_storagecluster_ceph_full_thresholds_params.py

```diff
@@ -0,0 +1,119 @@
+import logging
+import time
+
+import pytest
+
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.framework import config
+from ocs_ci.helpers.helpers import run_cmd_verify_cli_output
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier2,
+    skipif_external_mode,
+    brown_squad,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@tier2
+@brown_squad
+@skipif_external_mode
+@pytest.mark.polarion_id("OCS-XXX")
+class TestStorageClusterCephFullThresholdsParams(ManageTest):
+    """
+    Verify the ceph full thresholds storagecluster parameters move to cephcluster
+
+    """
+
+    @pytest.fixture(autouse=True)
+    def teardown_fixture(self, request):
+        """
+        Teardown function
+
+        """
+
+        def finalizer():
+            params_dict = {
+                "backfillFullRatio": "0.85",
+                "fullRatio": "0.8",
+                "nearFullRatio": "0.75",
+            }
+            self.set_storage_cluster_ceph_full_thresholds_params(params_dict)
+
+        request.addfinalizer(finalizer)
+
+    def test_storagecluster_ceph_full_thresholds_params(self):
+        """
+        Procedure:
+        1.Configure storagecluster CR
+        2.Wait 2 seconds
+        3.Verify ceph full thresholds parameters on cephcluster CR and storagecluster CR are same
+        4.Verify parameters with ceph api- 'ceph osd dump'
+        5.Configure the default params on storagecluster [treardown]
+
+        """
+        params_dict = {
+            "backfillFullRatio": "0.862",
+            "fullRatio": "0.793",
+            "nearFullRatio": "0.744",
+        }
```

Comment:
Thinkin about transforming to other format (list of dicts):
Move it to module level and reuse also this for validation.
```suggestion
        TRESHOLDS = [
            {
                "sc_key": "backfillFullRatio"
                "value": "0.862",
                "ceph_key": "backfillfull_ratio",
            },
            {
                "sc_key": "fullRatio"
                "value": "0.793",
                "ceph_key": "full_ratio",
            },
            {
                "sc_key": "nearFullRatio"
                "value": "0.744",
                "ceph_key": "nearfull_ratio",
            },
        ]
```

Reply:
done